indian_festivals/festival.py

```diff
@@ -24,10 +22,11 @@ class IndianFestivals(object):
     on yearly and monthly basis in json prettyprint format
     """
 
-    def __init__(self, year):
+    def __init__(self, year: int):
         # Parse below url to get all festivals and holidays info
-        page = urlopen(
-            "https://panchang-1205.appspot.com/calendars/indiancalendar?date=%s&lid=1261481&language=en" % year)
+        page = f"https://panchang.astrosage.com/calendars/indiancalendar?language=en&date={2022}"
```

Comment:
Don't hardcode the year here. It should be user defined

Reply:
Okay I will remove hard-code value

Reply:
Can we used ensure class to make sure that user will put integers value only ? 
If user put some string then it will rise an error.

Reply:
Yes. Please go ahead and add this check if possible in this PR

Reply:
for now, ensure not working with constructor so can we merge this PR itself.